snappy_pipeline/workflows/somatic_cnv_calling/__init__.py

```diff
@@ -0,0 +1,487 @@
+        self._validate_action(action)
+        return getattr(self, "_get_args_{}".format(action))
+
+    def _get_input_files_alleleCounter(self, wildcards: Wildcards) -> dict[str, str]:
+        ngs_mapping = self.parent.sub_workflows["ngs_mapping"]
+        base_path = "output/{mapper}.{library_name}/out/{mapper}.{library_name}".format(**wildcards)
+        input_files = {
+            "reference": self.w_config.static_data_config.reference.path,
+            "bam": ngs_mapping(base_path + ".bam"),
+            "bai": ngs_mapping(base_path + ".bam.bai"),
+            "loci": self.cfg.allele_counter.loci_prefix + f"{wildcards.chrom_name}.txt",
+        }
+        return input_files
+
+    def _get_args_alleleCounter(self, wildcards: Wildcards, input: InputFiles) -> dict[str, Any]:
+        args = {
+            "seed": self.cfg.seed,
+            "max-depth": self.cfg.allele_counter.max_depth,
+            "min-MQ": self.cfg.allele_counter.min_MQ,
+            "min-BQ": self.cfg.allele_counter.min_BQ,
+            "skip-any-set": self.cfg.allele_counter.skip_any_set.value,
+            "skip-any-unset": self.cfg.allele_counter.skip_any_unset.value,
+        }
+
+        return args
+
+    def _get_input_files_prepareHTS(self, wildcards: Wildcards) -> dict[str, str]:
+        input_files = {
+            "reference": self.w_config.static_data_config.reference.path,
+            "tumor_alleleFrequencies": [
+                "work/{mapper}.ascat.{tumor_name}/tmp/{library_name}_alleleFrequencies_chr{chrom_name}.txt".format(
+                    mapper=wildcards.mapper,
+                    tumor_name=wildcards.library_name,
+                    library_name=wildcards.library_name,
+                    chrom_name=chrom_name,
+                )
+                for chrom_name in self.chromosome_names
+            ],
+            "normal_alleleFrequencies": [
+                "work/{mapper}.ascat.{tumor_name}/tmp/{library_name}_alleleFrequencies_chr{chrom_name}.txt".format(
+                    mapper=wildcards.mapper,
+                    tumor_name=wildcards.library_name,
+                    library_name=self.parent.normals[wildcards.library_name],
+                    chrom_name=chrom_name,
+                )
+                for chrom_name in self.chromosome_names
+            ],
+        }
+
+        tpl = self.cfg.allele_counter.allele_prefix + "{chrom_name}.txt"
+        input_files["alleles"] = [
+            tpl.format(chrom_name=chrom_name, **wildcards) for chrom_name in self.chromosome_names
+        ]
+
+        if self.parent.libraryType != "WGS":
+            input_files["baits"] = self._get_target_interval_mapping_path(wildcards.library_name)
+
+        if self.config.sex.source == SexOrigin.AUTOMATIC:
+            input_files["sex"] = self._get_guess_sex_file(wildcards)
+
+        if self.cfg.allele_counter.path_probloci_file:
+            input_files["probloci_file"] = self.cfg.allele_counter.path_probloci_file
```

Comment:
```suggestion
        if f := self.cfg.allele_counter.path_probloci_file:
            input_files["probloci_file"] = f
```

Reply:
Yes, I should use the walrus more often...